sale_view_adj_sst/views/sale_view_adj.xml

```diff
@@ -0,0 +1,15 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<odoo>
+    <!-- Quote from CookBook PP.307-315 -->
```

Comment:
```suggestion
```

Reply:
Sorry, I'll be careful to delete comment out.